### PR TITLE
2023 tips update

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

Time to add, delete, and update tips!

## Why It's Good For The Game

TGMC always changes every year, and tips ought to reflect that. It's a community effort at best to ensure that other new or vet players (if they read tips) can gain a tidpit or two about TGMC.

## Changelog

:cl: novaepee, Al-1ce
spellcheck: 2023 tips update
/:cl:

### CODE REVIEW

**Comment by Mechswordsman** [comment]
Add one roundstart tip that is simple...

Pain.

**Comment by Ragecrabbo** [comment]
-As xeno you can freely melt barricades while in boiler or defiler gas as long as you can click on the right spot, the gas is also a really good hiding spot to launch ambushes from without marines being aware of you.
-Putting an instrument in your armor slot allows you to still be able to play it, charge into battle with song!
-Many maps spawn with free metal, plasteel and other important supplies in key spots. As a marine remember to grab them and as a xeno make sure to melt them.
-High Command doesn't want you to know. The runners on LV624 are free to take. I have 238 runners back at home.

File: strings/tips/meme.txt [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
+Tip: PFC Jim carries MRE, and so should you.
+Tip: The combat conga line is the ultimate power move.
+Tip: Pew pew makes beno big hurt.
+Tip: Specialists are removed, go ask requisitions for them.
+Tip: The TGMC Military Police did not do anything wrong.
+Tip: Renvagers are still invisible to this very day and likely they will remain unseen.
+Tip: We do not consult warcrimes here.
+Tip: As a xeno, rip and tear, until it is done.
+Tip: Unga dunga.
+Tip: Rouny exists.
```
Comment (psykzz) [inline strings/tips/meme.txt:33]: I think the point here was that these were not tips, but just statements.

**Comment by Lumipharon** [comment]
Do we have mech related tips?

**Review by psykzz — APPROVED** [review]

**Comment by Al-1ce** [comment]
Had some free-time. If even a quarter of these are used, it will add truckloads of variety.



>Marine


-There are water canteens in the vendors. Stay hydrated.
-If a corpsman is tending to you, try and hold still.
-Moving around with broken bones is dangerous.
-Jetpacks can fly over most things. Use them to escape your poor decisions.
-You can fit certain sidearms in your boots
-Ammo is plentiful. Bring a box with you.
-The RO works very hard. Be nice to them.
-When in doubt, blame the synth.
-While aiming, bipods and red-dot-sights allow you to shoot faster.
-Chemrette cigarettes can be very useful.
-Sleeping will slowly heal cloneloss.
-Xenos find revolvers very annoying.
-As a flamer, you are a very valuable target.
-Back fuel-tanks can be refilled via welding kits.
-The CSE can customize the detonation time of grenades.
-White Phosphorus is very dangerous.
-Your gloves can be made into fingerless ones with wirecutters.
-Batteries for energy weapons can be primed into makeshift grenades with the help of a multitool.
-Gigachads leave the autodoc on automatic.
-The awek plushy is not real. Stop asking about it.
-You can fit all sorts of things into your helmet.
-The time it takes for you to wield your gun can be the difference between life and death.
-Switching to your sidearm is faster than reloading.
-You can preform a tactical reload by dragging a fresh magazine from your belt towards your gun.
-With your sidearm in hand, RMB a pistol pouch to instantly reload. 
-You can turn the safety of your gun off.
-Smoke grenades can do more harm than good.
-A miner without a turret is a sitting duck.
-Claymores often 1-shot T1s.
-Crushers easily get trapped in razorwire.
-With the right tools, barricades can be moved.
-Do not under any circuimstances hug the rouny.
-Researchers can instruct Newt to follow them around.
-You can wear a facehugger as a mask.
-Dress to impress! Your fellow marines WILL notice how you look.
-You can wipe any bodypaint off your face with a piece of paper.
-People who wear green are usually trustworthy.
-The AI has a secret collection of USB sticks.
-Contrary to what the name may imply, "friendly-fire" is not very friendly at all.
-People who wear red are usually insane.
-A single protein bar will fill most people.
-Nanotrasen denies all involvement in the Great Rouny Massacre of 2433
-The Sons of Mars reside on a Martian analogue called Cydonia. They haven't lived on Mars in centuries.
-The USL stands for the United State of Lepidoptera, a fringe-terrorist group with connections to an alleged "moth-people".
-Humans love fire. Xenomorphs hate it.
-Stop picking your nose.
-All corpsmen are secretly changelings. Do not mind their stings.
-The CL can order pizza with their fax-machine.
-Insulting your fellow marines is terrible for morale. Be kind.
-The admins are _usually_ not out to get you.
-Very rarely, runners may evolve into something otherworldly.
-The most dangerous tactic is the unexpected.
-A single wraith can be fatal to the operation.
-Rail-mounted flashlights are generally useless.
-The Baldur armor module significantly increases your light output. Become the flashlight you always wanted to be!
-The Tyr module looks great on Xenonauten armor.
-Try not to bump into corpsmen. Your life might depend on it.
-There are rumors of people who use rightclick to examine tiles and move with arrow keys.
-Merging fancy lighting was a mistake.
-Right-Click functionality was forced on everyone until they developed stockholm-syrndrome for it.
-You can pat someone by targetting their head with help intent. 
-There is a way to hold hands via secret manouvers of the health-doll.
-Stop writing those weird flavortexts. You know who you are.
-There are stories of strange grey people who sometimes board vessels in distress. Surely nothing more than a rumor..
-Rattle me bones!
-With a cigarette pack in hand, target the mouth and click yourself to instantly take one.
-Marines have a hard time telling the difference between a costume and a real xenomorph.
-Requisition can buy yummy crayons for you to snack on.
-Do not eat the yellow snow.
-Spamming flares can be incredibly helpful.
-Traffic cones make nice hats.
-The cryopods in medbay heal most types of damage.
-When in doubt, throw them in the autodoc.
-Medbay may sometimes run out of power. To avoid this, scream at your nearest engineer to repair the fusion reactors.
-Doctor's Delight is one of the few healing chemicals that cannot be overdosed.
-Nanites are often not worth the risks.
-You can fit more than water in a canteen.
-15u from a standard inaprovaline injector will instantly heal 30% of all damage on someone in critical condition.
-There is a bazaar of weapons and tactics. Don't be afraid to find your niche!
-Always stick to your squad, or at least have a buddy.
-Walking to and from the FOB by yourself is an easy way to die.
-Xenomorphs are well armored. Over time, their exoskeleton will wear down, and they will be very vulnerable.
-The .410 MBX is the best shotgun in the game.
-You can attach a bullet charger to a BR-127 to become a discount autosniper.


>Xeno tips


-A large maze is almost always a good investment.
-Be clever with your walling. Try to allow for as much fluid movement as possible.
-Sticky resin can act as the path for your maze to be built around.
-The Hivemind is very skilled at flanking marines. Wait for them to push, and wall them off from behind!
-Praying to the Queen-Mother will usually net a response.
-Your queen is your lifeline. Try to listen to her.
-As a hunter, the movement of your Mirage is controlled by your intent.
-Defenders have a _very_ small window to sweep grenades back at marines with their tail.
-With ridiculously strong warding pheromones, bullets will completely bounce off a fortified defender.
-Aiming for the head is a good long-term investment.
-Aiming for the groin can easily damage someones liver.
-Arms and legs are particularly vulnerable to damage.
-To be robust is to know when to disengage.
-In a duel, observe your enemy. They are likely predictable.
-The QM has put a lot of faith into your hive. It would be a shame to let her down.
-Attack between the delay of a shotgun being fired.
-Listen to the sound of your enemies magazine! If it sounds hollow, it  will soon be empty.
-Strike hard and fast. 
-You are not disposable. However, sacrificing your life to save the queen is widely respected.
-A fortified defender is one of the best ways to stall a push.
-Skilled defilers can easily kill any marine in a span of seconds.
-Always have an escape plan.
-Tileswapping marines via help intent can have interesting consequences.
-As a boiler, you'll begin to glow with the more globules you have stored.
-If you see a red dot, RUN!
-Allowing marines to push caves only to block off their exit is hillariously effective.
-Escaping into a vent will save your life in almost every scenario.
-The shrike is usually bald.
-Afflictor is never coming back. Forget about it already.

**Comment by Al-1ce** [comment]
Had some free-time. If even a quarter of these are used, it will add truckloads of variety.



>Marine


-There are water canteens in the vendors. Stay hydrated.
-If a corpsman is tending to you, try and hold still.
-Moving around with broken bones is dangerous.
-Jetpacks can fly over most things. Use them to escape your poor decisions.
-You can fit certain sidearms in your boots
-Ammo is plentiful. Bring a box with you.
-The RO works very hard. Be nice to them.
-When in doubt, blame the synth.
-While aiming, bipods and red-dot-sights allow you to shoot faster.
-Chemrette cigarettes can be very useful.
-Sleeping will slowly heal cloneloss.
-As a mech-pilot, speed is not very important.
-As a mech-pilot, keep a close eye on your teamates. You lead the front!
-As a mech-pilot, make sure to repair and reload often.
-Xenos find revolvers very annoying.
-As a flamer, you are a very valuable target.
-Back fuel-tanks can be refilled via welding kits.
-The CSE can customize the detonation time of grenades.
-White Phosphorus is very dangerous.
-Your gloves can be made into fingerless ones with wirecutters.
-Batteries for energy weapons can be primed into makeshift grenades with the help of a multitool.
-Gigachads leave the autodoc on automatic.
-The awek plushy is not real. Stop asking about it.
-You can fit all sorts of things into your helmet.
-The time it takes for you to wield your gun can be the difference between life and death.
-Switching to your sidearm is faster than reloading.
-You can preform a tactical reload by dragging a fresh magazine from your belt towards your gun.
-With your sidearm in hand, RMB a pistol pouch to instantly reload. 
-You can turn the safety of your gun off.
-Smoke grenades can do more harm than good.
-A miner without a turret is a sitting duck.
-Claymores often 1-shot T1s.
-Crushers easily get trapped in razorwire.
-With the right tools, barricades can be moved.
-Do not under any circuimstances hug the rouny.
-Researchers can instruct Newt to follow them around.
-You can wear a facehugger as a mask.
-Dress to impress! Your fellow marines WILL notice how you look.
-You can wipe any bodypaint off your face with a piece of paper.
-People who wear green are usually trustworthy.
-The AI has a secret collection of USB sticks.
-Contrary to what the name may imply, "friendly-fire" is not very friendly at all.
-People who wear red are usually insane.
-A single protein bar will fill most people.
-Nanotrasen denies all involvement in the Great Rouny Massacre of 2433
-The Sons of Mars reside on a Martian analogue called Cydonia. They haven't lived on Mars in centuries.
-The USL stands for the United State of Lepidoptera, a fringe-terrorist group with connections to an alleged "moth-people".
-Humans love fire. Xenomorphs hate it.
-Stop picking your nose.
-All corpsmen are secretly changelings. Do not mind their stings.
-The CL can order pizza with their fax-machine.
-Insulting your fellow marines is terrible for morale. Be kind.
-The admins are _usually_ not out to get you.
-Very rarely, runners may evolve into something otherworldly.
-The most dangerous tactic is the unexpected.
-A single wraith can be fatal to the operation.
-Rail-mounted flashlights are generally useless.
-The Baldur armor module significantly increases your light output. Become the flashlight you always wanted to be!
-The Tyr module looks great on Xenonauten armor.
-Try not to bump into corpsmen. Your life might depend on it.
-There are rumors of people who use rightclick to examine tiles and move with arrow keys.
-Merging fancy lighting was a mistake.
-Right-Click functionality was forced on everyone until they developed stockholm-syrndrome for it.
-You can pat someone by targetting their head with help intent. 
-There is a way to hold hands via secret manouvers of the health-doll.
-Stop writing those weird flavortexts. You know who you are.
-There are stories of strange grey people who sometimes board vessels in distress. Surely nothing more than a rumor..
-Rattle me bones!
-With a cigarette pack in hand, target the mouth and click yourself to instantly take one.
-Marines have a hard time telling the difference between a costume and a real xenomorph.
-Requisition can buy yummy crayons for you to snack on.
-Do not eat the yellow snow.
-Spamming flares can be incredibly helpful.
-Traffic cones make nice hats.
-The cryopods in medbay heal most types of damage.
-When in doubt, throw them in the autodoc.
-Medbay may sometimes run out of power. To avoid this, scream at your nearest engineer to repair the fusion reactors.
-Doctor's Delight is one of the few healing chemicals that cannot be overdosed.
-Nanites are often not worth the risks.
-You can fit more than water in a canteen.
-15u from a standard inaprovaline injector will instantly heal 30% of all damage on someone in critical condition.
-There is a bazaar of weapons and tactics. Don't be afraid to find your niche!
-Always stick to your squad, or at least have a buddy.
-Walking to and from the FOB by yourself is an easy way to die.
-Xenomorphs are well armored. Over time, their exoskeleton will wear down, and they will be very vulnerable.
-The .410 MBX is the best shotgun in the game.
-You can attach a bullet charger to a BR-127 to become a discount autosniper.


>Xeno tips


-A large maze is almost always a good investment.
-Be clever with your walling. Try to allow for as much fluid movement as possible.
-Sticky resin can act as the path for your maze to be built around.
-The Hivemind is very skilled at flanking marines. Wait for them to push, and wall them off from behind!
-Praying to the Queen-Mother will usually net a response.
-Your queen is your lifeline. Try to listen to her.
-As a hunter, the movement of your Mirage is controlled by your intent.
-Defenders have a _very_ small window to sweep grenades back at marines with their tail.
-With ridiculously strong warding pheromones, bullets will completely bounce off a fortified defender.
-Aiming for the head is a good long-term investment.
-Aiming for the groin can easily damage someones liver.
-Arms and legs are particularly vulnerable to damage.
-To be robust is to know when to disengage.
-In a duel, observe your enemy. They are likely predictable.
-The QM has put a lot of faith into your hive. It would be a shame to let her down.
-Attack between the delay of a shotgun being fired.
-Listen to the sound of your enemies magazine! If it sounds hollow, it  will soon be empty.
-Strike hard and fast. 
-You are not disposable. However, sacrificing your life to save the queen is widely respected.
-A fortified defender is one of the best ways to stall a push.
-Skilled defilers can easily kill any marine in a span of seconds.
-Always have an escape plan.
-Tileswapping marines via help intent can have interesting consequences.
-As a boiler, you'll begin to glow with the more globules you have stored.
-If you see a red dot, RUN!
-Allowing marines to push caves only to block off their exit is hillariously effective.
-Escaping into a vent will save your life in almost every scenario.
-The shrike is usually bald.
-Afflictor is never coming back. Forget about it already.

**Comment by Al-1ce** [comment]
Had some free-time. If even a quarter of these are used, it will add truckloads of variety.



>Marine


-There are water canteens in the vendors. Stay hydrated.
-If a corpsman is tending to you, try and hold still.
-Moving around with broken bones is dangerous.
-Jetpacks can fly over most things. Use them to escape your poor decisions.
-You can fit certain sidearms in your boots
-Ammo is plentiful. Bring a box with you.
-The RO works very hard. Be nice to them.
-When in doubt, blame the synth.
-While aiming, bipods and red-dot-sights allow you to shoot faster.
-Chemrette cigarettes can be very useful.
-Sleeping will slowly heal cloneloss.
-As a mech-pilot, speed is not very important.
-As a mech-pilot, keep a close eye on your teamates. You lead the front!
-As a mech-pilot, make sure to repair and reload often.
-Xenos find revolvers very annoying.
-As a flamer, you are a very valuable target.
-Back fuel-tanks can be refilled via welding kits.
-The CSE can customize the detonation time of grenades.
-White Phosphorus is very dangerous.
-Your gloves can be made into fingerless ones with wirecutters.
-Batteries for energy weapons can be primed into makeshift grenades with the help of a multitool.
-Gigachads leave the autodoc on automatic.
-The awek plushy is not real. Stop asking about it.
-You can fit all sorts of things into your helmet.
-The time it takes for you to wield your gun can be the difference between life and death.
-Switching to your sidearm is faster than reloading.
-You can preform a tactical reload by dragging a fresh magazine from your belt towards your gun.
-With your sidearm in hand, RMB a pistol pouch to instantly reload. 
-You can turn the safety of your gun on.
-Smoke grenades can do more harm than good.
-A miner without a turret is a sitting duck.
-Claymores often 1-shot T1s.
-Crushers easily get trapped in razorwire.
-With the right tools, barricades can be moved.
-Do not under any circuimstances hug the rouny.
-Researchers can instruct Newt to follow them around.
-You can wear a facehugger as a mask.
-Dress to impress! Your fellow marines WILL notice how you look.
-You can wipe any bodypaint off your face with a piece of paper.
-People who wear green are usually trustworthy.
-The AI has a secret collection of USB sticks.
-Contrary to what the name may imply, "friendly-fire" is not very friendly at all.
-People who wear red are usually insane.
-A single protein bar will fill most people.
-Nanotrasen denies all involvement in the Great Rouny Massacre of 2433
-The Sons of Mars reside on a Martian analogue called Cydonia. They haven't lived on Mars in centuries.
-The USL stands for the United State of Lepidoptera, a fringe-terrorist group with connections to an alleged "moth-people".
-Humans love fire. Xenomorphs hate it.
-Stop picking your nose.
-All corpsmen are secretly changelings. Do not mind their stings.
-The CL can order pizza with their fax-machine.
-Insulting your fellow marines is terrible for morale. Be kind.
-The admins are _usually_ not out to get you.
-Very rarely, runners may evolve into something otherworldly.
-The most dangerous tactic is the unexpected.
-A single wraith can be fatal to the operation.
-Rail-mounted flashlights are generally useless.
-The Baldur armor module significantly increases your light output. Become the flashlight you always wanted to be!
-The Tyr module looks great on Xenonauten armor.
-Try not to bump into corpsmen. Your life might depend on it.
-There are rumors of people who use rightclick to examine tiles and move with arrow keys.
-Merging fancy lighting was a mistake.
-Right-Click functionality was forced on everyone until they developed stockholm-syrndrome for it.
-You can pat someone by targetting their head with help intent. 
-There is a way to hold hands via secret manouvers of the health-doll.
-Stop writing those weird flavortexts. You know who you are.
-There are stories of strange grey people who sometimes board vessels in distress. Surely nothing more than a rumor..
-Rattle me bones!
-With a cigarette pack in hand, target the mouth and click yourself to instantly take one.
-Marines have a hard time telling the difference between a costume and a real xenomorph.
-Requisition can buy yummy crayons for you to snack on.
-Do not eat the yellow snow.
-Spamming flares can be incredibly helpful.
-Traffic cones make nice hats.
-The cryopods in medbay heal most types of damage.
-When in doubt, throw them in the autodoc.
-Medbay may sometimes run out of power. To avoid this, scream at your nearest engineer to repair the fusion reactors.
-Doctor's Delight is one of the few healing chemicals that cannot be overdosed.
-Nanites are often not worth the risks.
-You can fit more than water in a canteen.
-15u from a standard inaprovaline injector will instantly heal 30% of all damage on someone in critical condition.
-There is a bazaar of weapons and tactics. Don't be afraid to find your niche!
-Always stick to your squad, or at least have a buddy.
-Walking to and from the FOB by yourself is an easy way to die.
-Xenomorphs are well armored. Over time, their exoskeleton will wear down, and they will be very vulnerable.
-The .410 MBX is the best shotgun in the game.
-You can attach a bullet charger to a BR-127 to become a discount autosniper.


>Xeno tips


-A large maze is almost always a good investment.
-Be clever with your walling. Try to allow for as much fluid movement as possible.
-Sticky resin can act as the path for your maze to be built around.
-The Hivemind is very skilled at flanking marines. Wait for them to push, and wall them off from behind!
-Praying to the Queen-Mother will usually net a response.
-Your queen is your lifeline. Try to listen to her.
-As a hunter, the movement of your Mirage is controlled by your intent.
-Defenders have a _very_ small window to sweep grenades back at marines with their tail.
-With ridiculously strong warding pheromones, bullets will completely bounce off a fortified defender.
-Aiming for the head is a good long-term investment.
-Aiming for the groin can easily damage someones liver.
-Arms and legs are particularly vulnerable to damage.
-To be robust is to know when to disengage.
-In a duel, observe your enemy. They are likely predictable.
-The QM has put a lot of faith into your hive. It would be a shame to let her down.
-Attack between the delay of a shotgun being fired.
-Listen to the sound of your enemies magazine! If it sounds hollow, it  will soon be empty.
-Strike hard and fast. 
-You are not disposable. However, sacrificing your life to save the queen is widely respected.
-A fortified defender is one of the best ways to stall a push.
-Skilled defilers can easily kill any marine in a span of seconds.
-Always have an escape plan.
-Tileswapping marines via help intent can have interesting consequences.
-As a boiler, you'll begin to glow with the more globules you have stored.
-If you see a red dot, RUN!
-Allowing marines to push caves only to block off their exit is hillariously effective.
-Escaping into a vent will save your life in almost every scenario.
-The shrike is usually bald.
-Afflictor is never coming back. Forget about it already.

**Comment by Al-1ce** [comment]
Had some free-time. If even a quarter of these are used, it will add truckloads of variety.



>Marine


-There are water canteens in the vendors. Stay hydrated.
-If a corpsman is tending to you, try and hold still.
-Moving around with broken bones is dangerous.
-Jetpacks can fly over most things. Use them to escape your poor decisions.
-You can fit certain sidearms in your boots
-Ammo is plentiful. Bring a box with you.
-The RO works very hard. Be nice to them.
-When in doubt, blame the synth.
-While aiming, bipods and red-dot-sights allow you to shoot faster.
-Chemrette cigarettes can be very useful.
-Sleeping will slowly heal cloneloss.
-As a mech-pilot, speed is not very important.
-As a mech-pilot, keep a close eye on your teamates. You lead the front!
-As a mech-pilot, make sure to repair and reload often.
-Xenos find revolvers very annoying.
-As a flamer, you are a very valuable target.
-Back fuel-tanks can be refilled via welding kits.
-The CSE can customize the detonation time of grenades.
-White Phosphorus is very dangerous.
-Your gloves can be made into fingerless ones with wirecutters.
-Batteries for energy weapons can be primed into makeshift grenades with the help of a multitool.
-Gigachads leave the autodoc on automatic.
-The awek plushy is not real. Stop asking about it.
-You can fit all sorts of things into your helmet.
-The time it takes for you to wield your gun can be the difference between life and death.
-Switching to your sidearm is faster than reloading.
-You can preform a tactical reload by dragging a fresh magazine from your belt towards your gun.
-With your sidearm in hand, RMB a pistol pouch to instantly reload. 
-You can turn the safety of your gun on.
-Smoke grenades can do more harm than good.
-A miner without a turret is a sitting duck.
-Claymores often 1-shot T1s.
-Crushers easily get trapped in razorwire.
-With the right tools, barricades can be moved.
-Do not under any circuimstances hug the rouny.
-Researchers can instruct Newt to follow them around.
-You can wear a facehugger as a mask.
-Dress to impress! Your fellow marines WILL notice how you look.
-You can wipe any bodypaint off your face with a piece of paper.
-People who wear green are usually trustworthy.
-The AI has a secret collection of USB sticks. What secrets do they hold?
-Contrary to what the name may imply, "friendly-fire" is not very friendly at all.
-People who wear red are usually insane.
-A single protein bar will fill most people.
-Nanotrasen denies all involvement in the Great Rouny Massacre of 2433
-The Sons of Mars reside on a Martian analogue called Cydonia. They haven't lived on Mars in centuries.
-The USL stands for the United State of Lepidoptera, a fringe-terrorist group with connections to an alleged "moth-people".
-Humans love fire. Xenomorphs hate it.
-Stop picking your nose.
-All corpsmen are secretly changelings. Do not mind their stings.
-The CL can order pizza with their fax-machine.
-Insulting your fellow marines is terrible for morale. Be kind.
-The admins are _usually_ not out to get you.
-Very rarely, runners may evolve into something otherworldly.
-The most dangerous tactic is the unexpected.
-A single wraith can be fatal to the operation.
-Rail-mounted flashlights are generally useless.
-You are living in a simulation.
-The Baldur armor module significantly increases your light output. Become the flashlight you always wanted to be!
-The Tyr module looks great on Xenonauten armor.
-Try not to bump into corpsmen. Your life might depend on it.
-There are rumors of people who use rightclick to examine tiles and move with arrow keys.
-Merging fancy lighting was a mistake.
-Right-Click functionality was forced on everyone until they developed stockholm-syrndrome for it.
-You can pat someone by targetting their head with help intent. 
-There is a way to hold hands via secret manouvers of the health-doll.
-Stop writing those weird flavortexts. You know who you are.
-There are stories of strange grey people who sometimes board vessels in distress. Surely nothing more than a rumor..
-Rattle me bones!
-With a cigarette pack in hand, target the mouth and click yourself to instantly take one.
-Marines have a hard time telling the difference between a costume and a real xenomorph.
-Requisition can buy yummy crayons for you to snack on.
-Do not eat the yellow snow.
-Spamming flares can be incredibly helpful.
-Traffic cones make nice hats.
-The cryopods in medbay heal most types of damage.
-When in doubt, throw them in the autodoc.
-Medbay may sometimes run out of power. To avoid this, scream at your nearest engineer to repair the fusion reactors.
-Doctor's Delight is one of the few healing chemicals that cannot be overdosed.
-Nanites are often not worth the risks.
-You can fit more than water in a canteen.
-15u from a standard inaprovaline injector will instantly heal 30% of all damage on someone in critical condition.
-There is a bazaar of weapons and tactics. Don't be afraid to find your niche!
-Always stick to your squad, or at least have a buddy.
-Walking to and from the FOB by yourself is an easy way to die.
-Xenomorphs are well armored. Over time, their exoskeleton will wear down, and they will be very vulnerable.
-The .410 MBX is the best shotgun in the game.
-You can attach a bullet charger to a BR-127 to become a discount autosniper.


>Xeno tips


-A large maze is almost always a good investment.
-Be clever with your walling. Try to allow for as much fluid movement as possible.
-Sticky resin can act as the path for your maze to be built around.
-The Hivemind is very skilled at flanking marines. Wait for them to push, and wall them off from behind!
-Praying to the Queen-Mother will usually net a response.
-Your queen is your lifeline. Try to listen to her.
-As a hunter, the movement of your Mirage is controlled by your intent.
-Defenders have a _very_ small window to sweep grenades back at marines with their tail.
-With ridiculously strong warding pheromones, bullets will completely bounce off a fortified defender.
-Aiming for the head is a good long-term investment.
-Aiming for the groin can easily damage someones liver.
-Arms and legs are particularly vulnerable to damage.
-To be robust is to know when to disengage.
-In a duel, observe your enemy. They are likely predictable.
-The QM has put a lot of faith into your hive. It would be a shame to let her down.
-Attack between the delay of a shotgun being fired.
-Listen to the sound of your enemies magazine! If it sounds hollow, it  will soon be empty.
-Strike hard and fast. 
-You are not disposable. However, sacrificing your life to save the queen is widely respected.
-A fortified defender is one of the best ways to stall a push.
-Skilled defilers can easily kill any marine in a span of seconds.
-Always have an escape plan.
-Tileswapping marines via help intent can have interesting consequences.
-As a boiler, you'll begin to glow with the more globules you have stored.
-If you see a red dot, RUN!
-Allowing marines to push caves only to block off their exit is hillariously effective.
-Escaping into a vent will save your life in almost every scenario.
-The shrike is usually bald.
-Afflictor is never coming back. Forget about it already.

**Comment by novaepee** [comment]
> Do we have mech related tips?

Not much.

File: strings/tips/xeno.txt [hunk 2]
```diff
@@ -38,3 +38,28 @@ As a spitter, boiler, praetorian, or the Queen take advantage of unguarded barri
...
+Defenders have a very small window to sweep grenades back at marines with their tail.
+Aiming for the head is a good long-term investment, causing brain damage on marines.
+Aiming for the groin can easily damage someones liver.
+Arms and legs are particularly vulnerable to damage.
+As a xeno, to be robust is to know when to disengage.
+As a xeno, in a duel, observe your enemy. They are likely predictable.
+The QM has put a lot of faith into your hive. It would be a shame to let her down.
+As a xeno, attack between the delay of a shotgun being fired.
+As a xeno, listen to the sound of your enemies magazine! If it sounds hollow, it will soon be empty.
+As a xeno, trike hard and fast.
```
Comment (Al-1ce) [inline strings/tips/xeno.txt:57]: It should be "**_strike_**" instead of trike.

File: strings/tips/xeno.txt [hunk 3]
```diff
@@ -38,3 +38,28 @@ As a spitter, boiler, praetorian, or the Queen take advantage of unguarded barri
 As a boiler or the Queen, you have the most powerful corrosive acid for melting things.
 As the Shrike, take advantage of your psychic abilities. Use Psychic Fling and Unrelenting Force to displace a marine or even push certain and hazardous objects to your enemies and vice versa.
 As the Queen, your screech does not only affect the mobs on view, but affects through walls or gases. However, the effect of the screech will be reduced.
+As a drone, hivelord, hivemind, shrike, and queen, large maze is almost always a good investment.
+As a drone, hivelord, hivemind, shrike, and queen, clever with your walling. Try to allow for as much fluid movement as possible.
```
Comment (Al-1ce) [inline strings/tips/xeno.txt:42]: There should be a linking verb here. "**_be_** clever with your walling." would suffice.

File: strings/tips/xeno.txt [hunk 4]
```diff
@@ -38,3 +38,28 @@ As a spitter, boiler, praetorian, or the Queen take advantage of unguarded barri
 As a boiler or the Queen, you have the most powerful corrosive acid for melting things.
 As the Shrike, take advantage of your psychic abilities. Use Psychic Fling and Unrelenting Force to displace a marine or even push certain and hazardous objects to your enemies and vice versa.
 As the Queen, your screech does not only affect the mobs on view, but affects through walls or gases. However, the effect of the screech will be reduced.
+As a drone, hivelord, hivemind, shrike, and queen, large maze is almost always a good investment.
```
Comment (Al-1ce) [inline strings/tips/xeno.txt:41]: Determiners come before the noun. "**_A_** large maze is almost always a good investment" would suffice.
- - -
I see room for another improvement. "As a drone, hivelord, hivemind, shrike, **_or_** queen," is easier to read.

File: strings/tips/xeno.txt [hunk 5]
```diff
@@ -38,3 +38,28 @@ As a spitter, boiler, praetorian, or the Queen take advantage of unguarded barri
...
+As a xeno, to be robust is to know when to disengage.
+As a xeno, in a duel, observe your enemy. They are likely predictable.
+The QM has put a lot of faith into your hive. It would be a shame to let her down.
+As a xeno, attack between the delay of a shotgun being fired.
+As a xeno, listen to the sound of your enemies magazine! If it sounds hollow, it will soon be empty.
+As a xeno, trike hard and fast.
+As a xeno, you are not disposable. However, sacrificing your life to save the queen is widely respected.
+A fortified defender is one of the best ways to stall a push.
+Skilled defilers can easily kill any marine in a span of seconds.
+As a xeno, lways have an escape plan.
```
Comment (Al-1ce) [inline strings/tips/xeno.txt:61]: Missing an **_"A"_** in "always."

File: strings/tips/meme.txt [hunk 6]
```diff
@@ -9,6 +9,13 @@ Tip: The ship AI takes song requests.
 Tip: Get gud.
 Tip: Dying is a skill issue.
 Tip: Being in crit state is a skill issue.
+Traffic cones make nice hats.
+Requisition can buy yummy crayons for you to snack on.
+There are stories of strange grey people who sometimes board vessels in distress. Surely nothing more than a rumor...
+Rattle me bones!
+There is a way to hold hands via secret manouvers of the health-doll.
+Stop writing those weird flavortexts. You know who you are
```
Comment (Al-1ce) [inline strings/tips/meme.txt:17]: Needs a period at the end of the sentence.

**Review by Al-1ce — CHANGES_REQUESTED** [review]
Grammar suggestions.

**Comment by Lumipharon** [comment]
@novaepee still going to add to this, or good to go?

**Comment by novaepee** [comment]
> @novaepee still going to add to this, or good to go?

Let me make some corrections, then it's good to go.

**Comment by Coastfront** [comment]
Violence is never the answer

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**